llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Transforms/Transforms.cpp

```diff
@@ -458,6 +458,22 @@ struct LinalgStrategyDecomposePass
                                                    linalg::Conv1DNwcWcfOp>,
              DownscaleSizeOneWindowed2DConvolution<linalg::Conv2DNchwFchwOp,
                                                    linalg::Conv1DNcwFcwOp>,
```

Comment:
Just use
```
linalg::populateDecomposeConvolutionPatterns(decompositionPattern);
```

This way you don't have to keep the files in sync, you just include the patterns as they are added.

Reply:
`populateDecomposeConvoluationPatterns` doesn't have a filter argument whereas this one has. We should add a filter to upstream and then change this. I can add a TODO.

Reply:
I updated to use `linalg::populateDecomposeConvolutionPatterns` because filter is not used during convolution.